src/game/client/view.h

```diff
@@ -223,6 +223,7 @@ class View : public SnapShot
     virtual View *Get_Next_View() { return m_next; }
     Coord3D &Get_Position() { return m_pos; }
     void Set_Position(const Coord3D *pos) { m_pos = *pos; }
+    bool World_To_Screen_Tri(const Coord3D *o1, ICoord2D *o2) { return World_To_Screen_Tri_Return(o1, o2) == 0; }
```

Comment:
Can be `const`?

Reply:
World_To_Screen_Tri_Return is not const so World_To_Screen_Tri can't be const.